osh/hub/osh_xmlrpc/scan.py

```diff
@@ -269,7 +269,7 @@ def find_tasks(request, query):
     if states:
         tasks = tasks.filter(state__in=states)
 
-    result = tasks.order_by("-dt_finished").values_list("id", flat=True)
```

Comment:
Thank for the quick fix, @rhyw!

Are we sure that the assignment of the `id` value is monotonic?  In other words, are we sure that the DB engine will not reuse previously deleted Task `id`s for new Tasks?  It may be safer to sort on `dt_created` instead.

@kdudka WDYT?

Reply:
@lzaoral I do not think we have ever deleted any tasks in OSH.  The retention policy deletes logs (scan results) associated with tasks but there is no code that would delete the task objects themselves.  I would prefer to sort by task ID because that is a plain integer.  Sorting by time would cause problems in case date/time was incorrectly set on OSH machines etc.

Reply:
Ok, but in that case, we should at least add a comment about this behaviour when tasks are deleted.

Reply:
By default the ID field is auto-incremental thus old IDs won't be re-used. I don't see any customization in kobo itself, thus I think it won't be a problem even if old tasks are deleted.

Reply:
TIL, @rhyw!  Ref: https://docs.djangoproject.com/en/5.0/topics/db/models/#automatic-primary-key-fields